httpie/downloads.py

```diff
@@ -407,26 +409,22 @@ def run(self):
 
     def report_speed(self):
 
-        now = time()
+        now = TIME_FUNC()
 
         if now - self._prev_time >= self._update_interval:
             downloaded = self.status.downloaded
-            try:
-                speed = ((downloaded - self._prev_bytes)
-                         / (now - self._prev_time))
-            except ZeroDivisionError:
-                speed = 0
+            speed = ((downloaded - self._prev_bytes)
+                     / (now - self._prev_time))
```

Comment:
`downloaded - self._prev_bytes` can be `0` if nothing new has been downloaded

Reply:
Yes, and it would not end on a `ZeroDivisionError`, it is safe.